osu.Desktop/Windows/WindowsKey.cs

```diff
@@ -2,10 +2,12 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using System;
+using System.Diagnostics.CodeAnalysis;
 using System.Runtime.InteropServices;
 
 namespace osu.Desktop.Windows
 {
+    [SuppressMessage("ReSharper", "IdentifierTypo")]
```

Comment:
We generally use comments instead (above the `namespace` line):

`// ReSharper disable IdentifierTypo`

Reply:
Does this also apply to the enum case? If so, I would have to add a `ReSharper restore` which is a first in this project.

<img width="355" alt="Screen Shot 2021-12-30 at 10 35 27 PM" src="https://user-images.githubusercontent.com/35318437/147807623-8e1aa227-ae53-452c-8723-d4324871222f.png">

Reply:
You can use ` // ReSharper disable once IdentifierTypo` here to disable once. Either above the line of above the `enum` definition.

Reply:
The last review wanted the whole enum to be disabled, see https://github.com/ppy/osu/pull/16273#discussion_r776220493.

Reply:
If it's not feasible to disable the inspection on the entire enum with a single line comment than I'm fine with disabling once on the item itself.